Documentation/concepts/overview.rst

```diff
@@ -52,7 +52,7 @@ Cilium Agent
 The Cilium agent (cilium-agent) runs on each Linux container host.  At a
 high-level, the agent accepts configuration that describes service-level
 network security and visibility policies.   It then listens to events in the
-container runtime to learn when containers are started or stopped, and it
+orchestration systems to learn when containers are started or stopped, and it
```

Comment:
Nitpicking, but this is not entirely correct: it gets info about start/stop via CNI.

Reply:
@brb I know but this is an overview explanation, not in detail.